### PR TITLE
Refine ProgramTranslator API English Doc for 2.0rc

### PR DESCRIPTION
**Pull request by zhhsplendid**
### PR types

Others
### PR changes

Docs
### Describe

Refine ProgramTranslator API Doc for 2.0RC. Screenshots:

image

### CODE REVIEW

File: python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py [hunk 1]
```diff
@@ -908,43 +905,43 @@ def func(x):
...
 
-                prog_trans = fluid.dygraph.ProgramTranslator()
 
-                x = np.ones([1, 2])
+                prog_trans = paddle.jit.ProgramTranslator()
+                x = paddle.ones([1, 2])
                 main_prog, start_prog, inputs, outputs = prog_trans.get_program(func, x)
                 print([i.name for i in inputs])
-                # ['feed_0'] the feed input variable name representing x
+                # [u'generated_tensor_0'] the feed input tensor name representing x
```
Comment (liym27) [inline python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py]: tensor -> Tensor
`Tensor` is a proper noun, better to capitalize its initial.

File: python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py [hunk 2]
```diff
@@ -908,43 +905,43 @@ def func(x):
...
                 print([o.name for o in outputs])
-                # ['_generated_var_4'] the fetch output variable name representing x_v        
+                # [u'_generated_var_4'] the fetch output tensor name representing x_v
```
Comment (liym27) [inline python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py]: the same as above

**Review by liym27 — COMMENTED** [review]
LGTM for the reset

File: python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py (hunk 1 above)
Comment (zhhsplendid) [reply python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py]: Done

File: python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py (hunk 2 above)
Comment (zhhsplendid) [reply python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py]: Done

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Review by liym27 — APPROVED** [review]

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**